src/ecrc-api/pom.xml

```diff
@@ -71,7 +71,11 @@
 			<artifactId>jaxb-impl</artifactId>
 			<version>2.2.11</version>
 		</dependency>
-
+		<dependency>
+			<groupId>io.jsonwebtoken</groupId>
+			<artifactId>jjwt</artifactId>
+			<version>0.9.1</version>
```

Comment:
Possibly break out version into POM properties?

Reply:
This is something we can discuss as I'm not sure what the best practice is.